### DISCUSSION TITLE
Support for setting a specific channel

### DISCUSSION THREAD

**Discussion by gavinhewitt** [spatie/laravel-slack-alerts#7] [Ideas]
Unless I missed something, there's seemingly no way to set the channel to which a message needs to be sent. Personally I would find that very useful.

**Comment by gavinhewitt** [comment]
Looked at the Slack docs and I think you need to create a per channel webhook, right?

**Comment by freekmurze** [comment]
There's some more info here: https://github.com/spatie/laravel-slack-alerts#using-multiple-webhooks

**Reply by gavinhewitt** [reply]
Thanks. I have read that but it wasn't immediately clear that that is the way to send messages to different channels. For instance, I did not know that you could have a webhook for a specific channel, but good to know!

**Comment by MrMooky** [comment]
Is it out of scope of this package to define a webhook url on the fly? For example when we want to send an alert to a user, who can store his webhook url in their settings. Something like this perhaps:

> SlackAlert::to(auth()->user()->slack_webhook_url)->message("Info");

**Reply by freekmurze** [reply]
If the code to achieve this is simple and testable, feel free to PR it 👍

**Reply by Nielsvanpach** [reply]
Added support for this in 1.0.3

**Reply by MrMooky** [reply]
Wow, that was fast. Really appreciated. 👍🏼
